R/get_airlines.R

```diff
@@ -16,12 +16,12 @@
 #' for weather data, \code{\link{get_airports}} for airport data, and 
 #' \code{\link{anyflights}} for a wrapper function  
 #' @export
-
+source("E:/GITHUB_Projects/anyflights/R/url_check.R")
```

Comment:
Looks like this `url_check` function is only stored locally. Could you add this function (and relevant documentation if need be) to the `R/` folder?

Reply:
Hi @simonpcouch, I just made another push with the `url_check.R` script.

Reply:
Great, thanks! This `source()` line will still error out on R-CMD-CHECK as it is platform-dependent--the script will be loaded by default on `devtools::load_all()`, so deleting this line should do the trick.

Reply:
I have deleted the `source()` line.